docs/Toy/toy/dialect.py

```diff
@@ -50,6 +53,16 @@ def verify_(self) -> None:
                 "Expected value and result types to be equal: "
                 f"{self.res.typ}, {self.value.type}")
 
+    def get_type(self) -> TensorTypeF64:
+        # Constant cannot be unranked
+        return cast(TensorTypeF64, self.value.type)
```

Comment:
Why this cannot happen exactly? Is this a problem in `DenseIntOrFPEElementsAttr`?

Reply:
Mostly because the way this code gets generated is in the upcoming `MLIRGen`, meaning from the parser, and we can only parse ranked tensors in the Toy source code.

Reply:
I mean also because by this point it's been verified, meaning we would've raise a `ValidationError` if it weren't.